src/main/resources/locales/en/translation/job-title.json

```diff
@@ -0,0 +1,12 @@
+{
+  "h1": "Employment details",
+  "p1": "Tell us some details about working for the organisation or person you’re making your claim against.",
+  "p2": "These details will be used to help progress your claim.",
+  "hint": "Tell us what role you had at the time of your dispute.",
+  "jobTitle": "Job title",
+  "errors": {
+    "jobTitle": {
+      "required": "Enter a job title"
```

Comment:
looks like job title is not a required field? can remove this error message as it'll never be used

Reply:
removed also